src/Servers/IIS/AspNetCoreModuleV2/IISLib/IISLib.vcxproj

```diff
@@ -78,11 +78,17 @@
     <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" Label="LocalAppDataPlatform" />
   </ImportGroup>
   <PropertyGroup Label="UserMacros" />
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <!-- Microsoft.Security.SystemsADM.10086 - SDL required warnings -->
+      <WarningLevel>Level4</WarningLevel>
```

Comment:
Can you try having IISLib import Build.Lib.Settings now after this PR I think you fixed all the main things that was blocking this when I was cleaning things up

Reply:
OK, done in my latest push. The shared settings have conformance mode on so I decided to fix those warnings too. So many unnecessary goto's 😆 . I want to do another cleanup pass on this but didn't want to make this PR too complex.